pom.xml

```diff
@@ -203,18 +202,16 @@
     <pluginManagement>
       <plugins>
         <plugin>
+          <!-- remove with future parent upgrade -->
           <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-site-plugin</artifactId>
-          <configuration>
-            <topSiteURL>scm:svn:https://svn.apache.org/repos/asf/maven/website/components/${maven.site.path}</topSiteURL>
-          </configuration>
+          <artifactId>maven-invoker-plugin</artifactId>
+          <version>3.4.0</version>
```

Comment:
This is technically not related to the parent upgrade, but this invoker version supports JDK 19 which is definitely nice. I could potentially split this into a different PR if needed.